app/models/waste_carriers_engine/order_item_log.rb

```diff
@@ -6,6 +6,7 @@
 module WasteCarriersEngine
   class OrderItemLog
     include Mongoid::Document
+    include CanCheckRegistrationStatus
 
     field :registration_id, type: BSON::ObjectId
```

Comment:
I think this can be dropped now as `belongs_to :registration` will make the id available.